docs/game-feedback-feature/SCREEN_RECORDING_PLAN.md

```
@@ -0,0 +1,408 @@
+
+const recorderOptions = {
+  mimeType: 'video/webm;codecs=vp9',
+  videoBitsPerSecond: 400_000, // 400 kbps for video
+  audioBitsPerSecond: 48_000,  // 48 kbps for audio
+};
+```
+
+### Browser API Usage
+
+**Screen Capture:**
+```typescript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)